src/sink.h

```diff
@@ -175,6 +175,11 @@ class Sink : public cyclus::Facility  {
                              "accept at each time step"}
   double capacity;
 
+  #pragma cyclus var {"default": "", "tooltip": "requested composition", \
+                      "doc": "name of recipe to use for material " \
+                             "requests"}
```

Comment:
I think that you should add a `"uitype": "recipe"` here. See http://fuelcycle.org/arche/cycpp.html#cycpp-table-1 for more details.